src/config/constants.js

```diff
@@ -70,7 +70,9 @@ export const DEFAULT_IMAGE_SRC =
 // time to be considered between 2 clicks for a double-click (https://en.wikipedia.org/wiki/Double-click#Speed_and_timing)
 export const DOUBLE_CLICK_DELAY_MS = 500;
 
+// root used for tree item menu
 export const ROOT_ID = 'root-id';
+export const SHARED_ROOT_ID = 'shared-root-id';
```

Comment:
style: are other IDs camelCase?

```suggestion
export const ROOT_ID = 'rootId';
export const SHARED_ROOT_ID = 'sharedRootId';
```